packages/compiler/src/render3/view/template.ts

```diff
@@ -1903,9 +1907,8 @@ export function parseTemplate(
   const {interpolationConfig, preserveWhitespaces} = options;
   const bindingParser = makeBindingParser(interpolationConfig);
   const htmlParser = new HtmlParser();
-  const parseResult = htmlParser.parse(
-      template, templateUrl,
-      {...options, tokenizeExpansionForms: true, leadingTriviaChars: LEADING_TRIVIA_CHARS});
+  const parseResult =
+      htmlParser.parse(template, templateUrl, {...options, tokenizeExpansionForms: true});
```

Comment:
To make it a default value here you can use a neat technique:

```suggestion
      htmlParser.parse(template, templateUrl, {leadingTriviaChars: LEADING_TRIVIA_CHARS, ...options, tokenizeExpansionForms: true});
```

Including it before `...options` means the value will be used if `options` doesn't override it. This makes `parseTemplate` more useful by itself (especially in tests) as users won't have to remember to configure it to behave exactly as it does when called from the real compiler.

Reply:
That's a lot nicer, you're right. Thanks!